src/NextGen/Gateways/PayPalCommerce/PayPalCommerceGateway.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Give\NextGen\Gateways\PayPalCommerce;
+
+use Give\Framework\EnqueueScript;
+use Give\Framework\PaymentGateways\Contracts\NextGenPaymentGatewayInterface;
+use Give\PaymentGateways\PayPalCommerce\Models\MerchantDetail;
+use Give\PaymentGateways\PayPalCommerce\PayPalCommerce;
+use Give\PaymentGateways\PayPalCommerce\Repositories\MerchantDetails;
+
+/**
+ * An extension of the PayPalCommerce gateway in GiveWP that supports the NextGenPaymentGatewayInterface.
+ */
+class PayPalCommerceGateway extends PayPalCommerce implements NextGenPaymentGatewayInterface
+{
```

Comment:
I forgot to tell you that you need to add this trait here or else the form will try to trigger a 301 redirect from the server.  I noticed this while testing. 
 This trait make sure it sends a json response instead.

```
use Give\NextGen\Framework\PaymentGateways\Traits\HandleHttpResponses;
```
```
use HandleHttpResponses;

Reply:
Resolved, see https://github.com/impress-org/givewp-next-gen/pull/139/commits/31afefb3e9a7464f9dbe0f79a366f3318e10bcea.